examples/text_to_image/requirements.txt

```diff
@@ -0,0 +1,6 @@
+accelerate
```

Comment:
add `bitsandbytes` for the 8bit adam option?

Reply:
`bitsandbytes` is optional and should be installed only when using `8-bit-adam`